### PR TITLE
(CDAP-7703) Reported operational stats for Yarn

### PR DESCRIPTION
**Pull request by bdmogal**
Jira: [CDAP-7703](https://issues.cask.co/browse/CDAP-7703)
Build: http://builds.cask.co/browse/CDAP-DUT5100

### CODE REVIEW

File: cdap-operational-stats-core/src/main/java/co/cask/cdap/operations/yarn/YarnInfo.java [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  }
+
+  /**
+   * Should only be called when HA is enabled.
+   */
+  private URL getHAWebURL() throws IOException {
+    InetSocketAddress activeRM = null;
+    Collection<String> rmIds = HAUtil.getRMHAIds(conf);
+    if (rmIds.isEmpty()) {
+      throw new IllegalStateException("");
```
Comment (anwar6953) [inline cdap-operational-stats-core/src/main/java/co/cask/cdap/operations/yarn/YarnInfo.java]: no exception message?

File: cdap-operational-stats-core/src/main/java/co/cask/cdap/operations/yarn/YarnApps.java [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+  }
+
+  @Override
+  public int getKilled() {
+    return killed;
+  }
+
+  @Override
+  public synchronized void collect() throws IOException {
+    reset();
```
Comment (anwar6953) [inline cdap-operational-stats-core/src/main/java/co/cask/cdap/operations/yarn/YarnApps.java:100]: does it seem like `reset` should be an interface method of `OperationalStats`?

Comment (bdmogal) [reply cdap-operational-stats-core/src/main/java/co/cask/cdap/operations/yarn/YarnApps.java:100]: not every stats needs it. but maybe we can do it later if a majority of them do?

**Comment by anwar6953** [comment]
LGTM

File: cdap-operational-stats-core/src/main/java/co/cask/cdap/operations/yarn/YarnApps.java (hunk 2 above)
Comment (anwar6953) [reply cdap-operational-stats-core/src/main/java/co/cask/cdap/operations/yarn/YarnApps.java:100]: sounds fair

**Comment by bdmogal** [comment]
last commit was only a message change, checkstyle passed after it. Squashed commits. Merging.

**Pull request merged**